File: iter3/python_development/Page/TranscriptPage.py
from iter3.python_development.Page.Page import Page
from iter3.python_development.Page.PageType import PageType
from iter3.python_development.System.FunctionType import FunctionType
from iter3.python_development.System.SystemMessage import SystemMessage


class TranscriptPage(Page):
    
    def __init__(self, content):
        super().__init__(content)
        self.setType("TRANSCRIPT_PAGE")
        self.setName("My Transcript Page")


    def runPage(self):
        self.showContent()
        self.takeInput()
        return SystemMessage("CHANGE_PAGE", "MAIN_MENU_PAGE", None)

File: src/Page/TranscriptPage.java
package Page;

import SystemMessage.FunctionType;
import SystemMessage.SystemMessage;

public class TranscriptPage extends Page {
	
	public TranscriptPage(String content) {
		super(content);
		setType(PageType.TRANSCRIPT_PAGE);
		setName("My Transcript Page");
	}
	
	@Override 
	public SystemMessage runPage() {
		showContent();
		takeInput();
		return new SystemMessage(FunctionType.CHANGE_PAGE, PageType.MAIN_MENU_PAGE, null);
	}
}
